src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,74 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Storage handles the reading and writing of data to text files.
+ *
+ * @author Jovyn Tan
+ * @version CS2103 AY21/22 Sem 1
+ */
+public class Storage {
+    private final String dataStoragePath;
+
+    /**
+     * A constructor for Duke Storage.
+     * @param dataStoragePath the path of the file where Duke data is stored.
```

Comment:
Perhaps can consider adding a line between the description and parameter section?